File: Python/3 Numeros em Ordem Crescente.py
aux = 0
n1 = input()
n2 = input()
n3 = input()

if (n1 > n2):
    aux = n2
    n2 = n1
    n1 = aux
if (n1 > n3 ):
    aux = n3
    n3 = n1
    n1 = aux
if (n2 > n3):
    aux = n3
    n3 = n2
    n2 = aux

print (n1)
print (n2)
print (n3)

File: C++/3 Numeros em Ordem Crescente.cpp
#include <iostream>
using namespace std;
int main() {
   int num_1, num_2,num_3, aux;
   cin >> num_1 >> num_2 >> num_3;
   while (( num_1 >num_2 ) || ( num_2 > num_3 )) {
      if (num_1 > num_2) {
         aux = num_1;
         num_1 = num_2;
         num_2 = aux;
      }
      if (num_2 > num_3) {
         aux = num_2;
         num_2 = num_3;
         num_3 = aux;
      }
   }
   cout << num_1 << endl << num_2 << endl << num_3 << endl;
   return 0;
}
